sdk/ai/azure-ai-generative/azure/ai/generative/evaluate/metrics/_custom_metric.py

```diff
@@ -2,22 +2,16 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
 import abc
-import importlib
-import json
-import os
-import urllib.parse
-
-from azure.ai.generative.evaluate.metrics._aggregators import mean
-from azure.ai.generative.evaluate.metrics._parsers import ScoreReasonParser, ScoreParser
 
 
 class Metric(metaclass=abc.ABCMeta):
 
-    def __init__(self, name):
+    def __init__(self, *, name, description, **kwargs):
         self.name = name
+        self.description = description
 
 
-class CodeMetric(Metric):
+class _CodeMetric(Metric):
     """Evaluation code metric
 
     :keyword name: Name of the metric.
```

Comment:
I thought the review feedback is to allow custom functions to accept a flat list of kwargs? Can you confirm?

Reply:
Yes it is this is just internal class.